src/main.js

```diff
@@ -739,6 +739,15 @@ function initializeAfterAppReady() {
       defaultPath: os.homedir() + '/Downloads/' + filename,
       filters,
     });
+
+    item.on('done', (doneEvent, state) => {
+      if (state === 'completed') {
+        mainWindow.webContents.send('download-complete', {
```

Comment:
if you create a custom channel, you can add all the info that requires the display format (see jira ticket for that)

you probably need to get that info from the webcontents object in the `will-download` event, that way you can figure out which server initiated the download process.